tests/internal/data/config_format/convert/fb-parser-conf2yaml.pl

```diff
@@ -0,0 +1,492 @@
+#!/usr/bin/env perl
```

Comment:
This is just my opinion but it would be nice to be written down in Python instead of Perl...

Reply:
> This is just my opinion but it would be nice to be written down in Python instead of Perl...

I'm sure it's not just your opinion, probably shared by others :)

But I'm about 10x more proficient in perl than python. This tool probably won't be needed longterm, once the current parser files are translated (but I added it to show my work).

I'm sure it could be translated/rewritten into python mostly automatically, but if so I wouldn't be qualified to debug it.